Cargo.toml

```diff
@@ -75,6 +75,9 @@ csfle = ["mongocrypt", "rayon", "num_cpus"]
 # TODO: pending https://github.com/tokio-rs/tracing/issues/2036 stop depending directly on log.
 tracing-unstable = ["tracing", "log"]
 
+[patch."https://github.com/mongodb/bson-rust"]
```

Comment:
This won't get merged, it's just needed until https://github.com/mongodb/bson-rust/pull/382 goes in.